### DISCUSSION TITLE
node types not shown

### DISCUSSION THREAD

**Discussion by bobblkabb** [sayanarijit/xplr#491] [Q&A]
Downloaded xplr and copied it to /usr/bin and copied init.lua from the github repo to ~/.config/xplr.
I'm trying to get the icons, like pdf, zip and so on but nothing changes. Always the same odd symbols.

What's going wrong? How can I fix this?

**Comment by sayanarijit** [comment]
Hi, you will need to install a plugin like [icons.xplr](https://github.com/prncss-xyz/icons.xplr) or configure the icons yourself as done [here](https://github.com/prncss-xyz/icons.xplr/blob/main/init.lua).

**Reply by sayanarijit** [reply]
See https://xplr.dev/en/awesome-plugins#theme

**Comment by bobblkabb** [comment]
Thank you! Looking better but still missing a few icons like 'mp4', 'pdf'. Which nerd font do I need?
[https://www.nerdfonts.com/font-downloads](https://www.nerdfonts.com/font-downloads)

**Reply by sayanarijit** [reply]
Any should work fine... I'm using `Hack`.

**Comment by leakedmemory** [comment]
I'm having the same problem, but in my case some nodes work and others don't. Some files appears as an "f" and all the directories as deltas.

I'm using JetBrains Mono, [hartan/web-devicons.xplr](https://gitlab.com/hartan/web-devicons.xplr) and xplr version is 0.21.7.

![2024-04-19_12-19](https://github.com/sayanarijit/xplr/assets/76412614/60720633-eca6-4bbd-9ee8-18855f691abc)

**Reply by PhML** [reply]
@leakedmemory I have the same problem. It seems to be the default icons: https://github.com/sayanarijit/xplr/blob/main/src/init.lua#L780

As explained by @sayanarijit in https://github.com/sayanarijit/xplr/discussions/491#discussioncomment-2979086, I added this to my configuration:

```lua
local function blue(x)
	return "\x1b[34m" .. x .. "\x1b[0m"
end
local function magenta(x)
	return "\x1b[35m" .. x .. "\x1b[0m"
end

local function cyan(x)
	return "\x1b[36m" .. x .. "\x1b[0m"
end
xplr.config.node_types.directory.meta.icon = blue("")
xplr.config.node_types.file.meta.icon = cyan("")
xplr.config.node_types.symlink.meta.icon = magenta("")
```

But I don’t know if it is normal to have to do it manually.
